### PR TITLE
LPS-174056 [Raylife AP] Enable Account Functionality for User Avatar

### PR DESCRIPTION
**Pull request by raquelpaiva**
[LPS174056](https://issues.liferay.com/browse/LPS-174056)

### CODE REVIEW

**Comment by joanagrego** [comment]
Hi @raquelpaiva, please rename your branch to _LPS-174056_ and your PR to _LPS-174056 [Raylife AP] Enable Account Functionality for User Avatar_

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+};
+
+// console.log(Liferay.ThemeDisplay);
+
+// const redirectUrlWithoutAp = (portletId) => {
+// 	const token = Liferay.authToken;
+// 	const routeconfig = '&p_p_lifecycle=0&p_p_auth=';
+
+// 	window.location.href = `${portletId}${routeconfig}${token}`;
+// };
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.js:50]: Please delete it.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+ */
+
+const avatarElement = document.querySelector('.applications-menu-header span');
+const left = avatarElement.offsetLeft;
+const height = avatarElement.offsetHeight;
+
+const dropdown = document.querySelector('.dropdown.applications-menu-wrapper');
+dropdown.onclick = () => dropdown.classList.toggle('show');
+
+// const dropdownItems = document.querySelector('.dropdown-items');
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.js:23]: Please delete it.

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+const userId = Liferay.ThemeDisplay.getUserName();
+document.querySelector('.user-name').innerHTML = userId;
+
+// console.log('user', userId);
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.js:28]: Please delete it.

**Review by joanagrego — CHANGES_REQUESTED** [review]

**Comment by joanagrego** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.js (hunk 1 above)
Comment (raquelpaiva) [reply modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-fragments/fragments/avatar-dropdown-menu/index.js:50]: Done

**Pull request closed without merging**